mkosi/util.py

```diff
@@ -59,6 +59,12 @@ def round_up(x: int, blocksize: int = 4096) -> int:
     return (x + blocksize - 1) // blocksize * blocksize
 
 
+def string_strip_prefix(s: str, prefix: str) -> Optional[str]:
+    if s.startswith(prefix):
+        return s.removeprefix(prefix)
+    return None
```

Comment:
I would name this function `startswith` so it mimicks the same function we have in systemd